### ISSUE TITLE
The lack of explicit connection between webhooks and workflows makes webhook errors hard to diagnose

### ISSUE THREAD

**Issue by grahamalama** [mozilla/jira-bugzilla-integration#567]
In Bugzilla, up to this point, we've created a new webhook per product. We have a naming convention of `JBI-<environment>-<product>`. For example:

| Name | Product | Component |
|--------|--------|--------|
| JBI-PROD-Core | Core | ANY |
| JBI-PROD-DataPlatformAndTools | Data Platform and Tools | ANY |
| JBI-PROD-DevTools | DevTools | ANY |

In config, projects specify the Jira project they map to and what whiteboard tag they use. There is no information indicating which product or products a workflow uses. 

This makes webhook errors hard to diagnose. For example, if I were to log in and see that the `JBI-PROD-Core` webhook had errors piling up and was disabled, I wouldn't have the information to know which project likely caused that blockage at a glance.

**Comment by leplatrem** [comment]
Should we add the webhook name do the `ActionContext`?  It would allow us to query BigQuery at least

**Comment by leplatrem** [comment]
Is this relevant if we do #21 ?

**Comment by alexcottner** [comment]
I don't think this is relevant now that we have 1 webhook for any project, and the retry queue.
We have some pretty good logging in the retry process and sentry alerting from the web service now.
Going to mark this as closed.
